virttest/qemu_vm.py

```diff
@@ -2779,9 +2781,9 @@ def create(self, name=None, params=None, root_dir=None,
                         nic.vhostfds = ':'.join(vhostfds)
                         for fd in vhostfds:
                             pass_fds.append(int(fd))
-                    elif nic.nettype == 'user':
+                    elif nic.nettype == 'user' or nic.nettype == 'socket':
                         logging.info("Assuming dependencies met for "
-                                     "user mode nic %s, and ready to go"
+                                     "user/socket mode nic %s, and ready to go"
```

Comment:
Or `  "%s mode nic %s, and ready to go" % (nic.nettype, nic.nic_name)` is more accurate.